### ISSUE TITLE
[Question]: How to get the right BrowserType from a device name? 

### ISSUE THREAD

**Issue by Rafiot** [microsoft/playwright-python#1496]
### Your question

I noticed that the CLI is able to figure out the right `BrowserType` to use when it is launched from the command line:

```
playwright open --device="Desktop Safari" wikipedia.org  # Webkit
playwright open --device="Desktop Firefox" wikipedia.org  # Firefox
playwright open --device="Desktop Chrome" wikipedia.org  # Chrome
``` 

But [the documentation](https://playwright.dev/python/docs/api/class-playwright#playwright-devices) seems to say that I have to initialize a `BrowserType` before I can pass the settings to the context, which partially defeats the purpose of the device settings.

I can implement my own logic do initialize the right `BrowserType` for each device, but as `playwright open` can already do that, that seems superfluous.

**Comment by rwoll** [comment]
You can get it from the `devices` object: https://playwright.dev/python/docs/emulation#devices. It's a property in the dict returned when you do `playwright.devices["Desktop Safari"]`

**Comment by Rafiot** [comment]
I can get the settings, but I still need to initialize a browser (`browser = await playwright.webkit.launch(headless=False)`), and then initialize the context (`context = await browser.new_context(**pixel_2)`). 

In my case, I want to start from the device name (say `'Pixel 2'`) and get the right `BrowserType`. And afaik, I can't figure that out without parsing the user agent, or mapping `'Pixel 2'` to `webkit`.

**Comment by rwoll** [comment]
Gotcha—looks like we are missing the defaultBrowserType property in the devices entries that are available in Node and could be used to programmatically pick `playwright.{webkit,chromium,firefox}`.

**Comment by Rafiot** [comment]
\o/ nice, thank you!

**Comment by rwoll** [comment]
For now, you can get the defaultDeviceType from https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/server/deviceDescriptorsSource.json, but it should be exposed on `playwright.devices['Pixel 2']` in the future.

**Comment by Rafiot** [comment]
Excellent, I'll use the dump for now and switch to that when available.

**Comment by rwoll** [comment]
@Rafiot Just shipped in v1.25.0 🎉 ! You can use it like the test does: https://github.com/microsoft/playwright-python/pull/1497/files

**Comment by Rafiot** [comment]
Nice, thanks! And thinks for the hint, I didn't know I could get a browser like that: `playwright[device_type]`
